docs/01-app/03-api-reference/04-functions/generate-metadata.mdx

```diff
@@ -1170,33 +1172,33 @@ There are two default `meta` tags that are always added even if a route doesn't
 
 ### Streaming metadata
 
-Metadata returned by `generateMetadata` is streamed to the client. This allows Next.js to inject metadata into the HTML as soon as it's resolved.
+Streaming metadata allows Next.js to render and send the initial UI to the browser, without waiting for `generateMetadata` to complete.
 
-Streamed metadata is appended to the `<body>` tag. Since metadata mainly targets bots and crawlers, Next.js streams metadata for bots that can execute JavaScript and inspect the full DOM (e.g. `Googlebot`). We have verified that these bots interpret the metadata correctly.
+When `generateMetadata` resolves, the resulting metadata tags are appended to the `<body>` tag. We have verified that metadata is interpreted correctly by bots that execute JavaScript and inspect the full DOM (e.g. `Googlebot`).
 
-For **HTML-limited** bots that can’t run JavaScript (e.g. `Twitterbot`), metadata continues to block page rendering and is placed in the `<head>` tag.
+For **HTML-limited bots** that can’t execute JavaScript (e.g. `Twitterbot`, `Slackbot`, `Bingbot`), metadata continues to block page rendering. The resulting metadata will be available in the `<head>` tag.
```

Comment:
did we confirm these bots struggle with JS execution? I was only ever aware of facebookexternalhit but I assume we did verify these other bots have the same issue?

Reply:
In the spirit of keeping the change set small, and not make claims about bots we haven't tested (I checked Slackbot and it seems to work fine) - I'll keep Twitterbot only, we can come back to this later on after doing some more verification.